### ISSUE TITLE
"He'll" being marked as profane word

### ISSUE THREAD

**Issue by ngan** [mobomo/profanity_filter#6]
Does it make sense for `he'll` to be marked as a profane word?

**Comment by adambair** [comment]
No, not at all. This is totally a bug. Is this new or has it always been this way -- I accepted a pull request which modified the dictionary which may have introduced this -- I'll check it out and see what's up.

**Comment by ngan** [comment]
I'm on gem version 0.0.2.  Not sure if it's new or not.

**Comment by adambair** [comment]
I released 0.1.1 last week, let me know if you still have trouble after the upgrade (or if need it to work in this specific version).

**Comment by ngan** [comment]
Yea, I saw the new versions, but haven't had time to upgrading.  But I also noticed that two changes (from the two versions diff) doesn't touch the process of checking word's profanity at all.  The problem is the regex: `\W` considers `'` as non-word, so `he'll` gets turned into `hell`.  I think the best thing is to use `\s` to swallow just spaces instead.

**Comment by ngan** [comment]
Any update on this?
